ipstools/synopsys_defines.py

```diff
@@ -25,3 +25,6 @@
 SYNOPSYS_ADD_IPS_FILES_CMD = "source scripts/analyze/ips/%s\n"
 SYNOPSYS_ADD_RTL_FILES_CMD = "source scripts/analyze/rtl/%s\n"
 
+SYNOPSYS_INC_DIRS_PREAMBLE  = "set synthesis_include_paths [ list \"%s/%s/include\" \\\n"
```

Comment:
`include` should be `includes`. If you check the `rtl` subdirectory, you see the folder is called `includes`